test/e2e/.rake_tasks~

```diff
@@ -3,6 +3,7 @@ clean_logs
 fixture_wallets_decode
 fixture_wallets_encode
 fixture_wallets_template
+get_docker_logs
```

Comment:
What is this file?
I am used to the convention where files ending with `~` are backup files.
Is it hand-crafted, or auto-generated from `Rakefile`?
If the latter, then it should be removed and added to .gitignore.

Reply:
It was autogenerated first but I've been adding new tasks to it. As far as I see rake uses it for autocompletion so it seemed useful to me to have it checked in, although it looks a bit awkward. There is another one .rakeTasks which seems to be autogenerated by Atom, and it's not useful. I'll remove that one.

Reply:
It was autogenerated first but I've been adding new tasks to it. As far as I see rake uses it for autocompletion so it seemed useful to me to have it checked in, although it looks a bit awkward. There is another one .rakeTasks which seems to be autogenerated by Atom, and it's not useful. I'll remove that one.